apps/scan/backend/src/scanners/pdi/state_machine.ts

```diff
@@ -850,7 +850,10 @@ function buildMachine({
           on: {
             SCANNER_STATUS: [
               {
-                cond: (_, { status }) => !status.documentJam,
+                cond: (_, { status }) =>
+                  !status.documentJam &&
+                  !anyRearSensorCovered(status) &&
+                  !anyFrontSensorCovered(status),
```

Comment:
Since we're declaring a jam above when we detect a paper in the rear even though the scanner itself may not have set the `documentJam` flag, we need to make sure the scanner is entirely clear of paper before we exit the jammed state. With the previous approach, we would exit the jammed state immediately since no `documentJam` flag was set.